doc/sr_cpp_walkthrough.rst

```diff
@@ -3,21 +3,19 @@ C++
 ***
 
 
-In this section, examples are presented using the SmartRedis C++
-API to interact with the RedisAI tensor, model, and script
-data types.  Additionally, an example of utilizing the
-SmartRedis ``DataSet`` API is also provided.
+This section presents examples that use the SmartRedis C++ API to 
+interact with the RedisAI tensor, model, and script data types. 
+Additionally, this section demonstrates an example of utilizing the SmartRedis ``DataSet`` API.
 
 
 
 .. note::
       The C++ API examples rely on the ``SSDB`` environment
-      variable being set to the address and port of the Redis database.
+      variable set to the address and port of the Redis database.
```

Comment:
how about "having been set" -- what we're relying on here is that the variable has to be set before we can proceed

Reply:
Getting rid of passive tense is definitely a good thing.  Yes, SSDB has to be manually set by the user (if they set up their own Redis cluster), but it's automatically set up by SmartSim when you go through the launcher.

Reply:
Is it worth it adding this short info? -> "If a User sets up their own Redis cluster, then they must manually set the address and port of their Redis database, otherwise it is automatically set through the launcher"

Reply:
I think I was being extra, and also I am having trouble thinking of another way to put it without it turning into something super wordy